internal/server/volume/server.go

```diff
@@ -0,0 +1,144 @@
+package volume
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/klog"
+
+	"github.com/kubernetes-csi/csi-proxy/client/apiversion"
+	"github.com/kubernetes-csi/csi-proxy/internal/server/volume/internal"
+)
+
+type Server struct {
+	hostAPI API
+}
+
+type API interface {
+	ListVolumesOnDisk(diskID string) (volumeIDs []string, err error)
+	// MountVolume mounts the volume at the requested global staging path
+	MountVolume(volumeID, path string) error
+	// DismountVolume gracefully dismounts a volume
+	DismountVolume(volumeID, path string) error
+	// IsVolumeFormatted checks if a volume is formatted with NTFS
```

Comment:
is there a fstype parameter somewhere, if it's empty, then it's NTFS

Reply:
Yes, a few more parameters have to be added. Will do a follow up PR with that:
https://github.com/kubernetes-csi/csi-proxy/pull/17#issuecomment-571007850